SSD/ssd_cmd_buffer_control_test.cpp

```diff
@@ -6,39 +6,77 @@ using namespace testing;
 
 class BufferControlFixture : public Test {
 public:
-  CmdBufferControl &cmdBuffer = CmdBufferControl::getInstance    ();
-
+  void SetUp() override { cmdBuffer.clearAllBuffer(); }
+  CmdBufferControl &cmdBuffer = CmdBufferControl::getInstance();
   const std::string BUFFER_EMPTY = "1_empty,2_empty,3_empty,4_empty,5_empty,";
 };
 
 TEST_F(BufferControlFixture, BufferCheckAfterInit) {
   std::string expected_ret = BUFFER_EMPTY;
-
   EXPECT_EQ(expected_ret, cmdBuffer.getBufferNameList());
 }
 
 TEST_F(BufferControlFixture, BufferUpdateByIndexTest) {
-  std::string expected_ret = "1_empty,2_W_20_0xABCDABCD,3_empty,4_empty,5_empty,";
-  cmdBuffer.updateBufferByIndex(2, "2_W_20_0xABCDABCD");
+  std::string expected_ret =
+      "1_empty,2_W_20_0xABCDABCD,3_empty,4_empty,5_empty,";
+  cmdBuffer.updateBufferByIndex(2, "W_20_0xABCDABCD");
   EXPECT_EQ(expected_ret, cmdBuffer.getBufferNameList());
 }
 
 TEST_F(BufferControlFixture, BufferUpdateByIndex2Test) {
-  EXPECT_TRUE(cmdBuffer.updateBufferByIndex(2, "2_W_20_0xABCDABCD"));
-  EXPECT_TRUE(cmdBuffer.updateBufferByIndex(2, "2_R_10"));
+  EXPECT_TRUE(cmdBuffer.updateBufferByIndex(2, "W_20_0xABCDABCD"));
+  EXPECT_TRUE(cmdBuffer.updateBufferByIndex(2, "R_10"));
 }
```

Comment:
혹시 Write 가 Error 인 경우는 테스트 안해봐도 괜찮나요?

```suggestion
  EXPECT_TRUE(cmdBuffer.updateBufferByIndex(2, "W_100_0xABCDABCD"));
  EXPECT_TRUE(cmdBuffer.updateBufferByIndex(2, "R_10"));
}
```

Reply:
현재 buffer쪽 file update에서는 이전 ssd Driver 에서 range check 잘되고 잘 정리된 String이 내려왔을거라 생각하여.
따로 예외처리는 하지 못했습니다.
내부 의견 확인 후 필요시 추후 적용하도록 하겠습니다.

Reply:
아 잘 정리된 인자값을 받는 거라면 예외 없어도 괜찮을 거 같아요!